### ISSUE TITLE
Duplikate aus Tatzeit-Vorschlägen entfernen

### ISSUE THREAD

**Issue by RalphBike** [weg-li/weg-li#412]
Die Vorschläge für die Tatzeiten sind super. Allerdings sollten sie keine Duplikate enthalten, wie 2x "18:24"

**Comment by phoet** [comment]
Ja, das ist mir auch schon aufgefallen, kommt aber daher, dass die Uhrzeiten keine Duplikate sind, sondern einen unterschiedlichen Zeitstempel haben der in der selben Minute liegt.

**Comment by RalphBike** [comment]
Jupp, klar. Tritt auch nur bei mehreren Fotos auf.

**Comment by phoet** [comment]
Ist das Probem noch vorhanden?

**Comment by RalphBike** [comment]
Ja. Z. B. in 1c2acb0dc23ef0f23f4c31a16fd858cc
